{
    "checksum": "297790b4ed8a6c3f5f9d1e05a3d1e2dd78f027141eccee7d03bb4204293ea623",
    "file": "errorcodes.py",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/x-python"
    },
    "number_of_duplicates": 1,
    "package": "python-psycopg2cffi",
    "path": "python-psycopg2cffi/2.8.1-2/psycopg2cffi/errorcodes.py",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 0,
        "license": "/copyright/license/python-psycopg2cffi/2.8.1-2/",
        "metric": {
            "size": 760
        },
        "pts_link": "https://tracker.debian.org/pkg/python-psycopg2cffi",
        "sloc": [
            [
                "python",
                10515
            ],
            [
                "makefile",
                17
            ]
        ],
        "suites": [
            "bookworm",
            "bullseye",
            "sid"
        ],
        "vcs_browser": "https://salsa.debian.org/openstack-team/python/python-psycopg2cffi",
        "vcs_type": "git"
    },
    "raw_url": "/data/main/p/python-psycopg2cffi/2.8.1-2/psycopg2cffi/errorcodes.py",
    "stat": {
        "perms": "rw-r--r--",
        "size": 13042,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "2.8.1-2"
}